example/index.js

```diff
@@ -38,17 +40,35 @@ class MainApp extends React.Component {
         <TouchableOpacity onPress={this.showFullScreenAd}>
           <Text>Show interstitial ad</Text>
         </TouchableOpacity>
-      </View>
+        <BannerView
+          style={styles.banner50}
+          placementId="1912255062335197_1915775421983161"
+          onAdPress={this.onBannerPress}
+          onAdError={this.onBannerError}
+        />
+        <BannerView
+          type="large"
+          style={styles.banner90}
+          placementId="1912255062335197_1954647211429315"
+          onAdPress={this.onBannerPress}
```

Comment:
Maybe simpler to use `onPress` and `onError` instead of `onAdPress` and `onAdError`